src/admin-events/admin-events.spec.js

```diff
@@ -0,0 +1,106 @@
+const ethers = require('ethers');
+const { getAddress } = require('@uma/contracts-node');
+
+// Load Config files
+const { TransactionEvent } = require('forta-agent');
+
+const chainId = 1;
+const votingAddressPromise = getAddress('Voting', chainId);
+
+const { createAlert, handleTransaction } = require('./admin-events');
+
+// TransactionEvent(type, network, transaction, receipt, traces, addresses, block)
+function createTxEvent({ logs, addresses }) {
+  return new TransactionEvent(null, null, null, { logs }, [], addresses, null);
+}
+
+// tests
+describe('admin event monitoring', () => {
+  describe('handleTransaction', () => {
+    it('returns empty findings if contract address does not match', async () => {
+      // logs data for test case:  no address match + no topic match
+      const logsNoMatchAddress = [
+        {
+          address: ethers.constants.AddressZero,
+          topics: [
+            ethers.constants.HashZero,
+          ],
+        },
+      ];
+
+      // build txEvent
+      const txEvent = createTxEvent({
+        logs: logsNoMatchAddress,
+        addresses: { [ethers.constants.AddressZero]: true },
+      });
+
+      // run agent
+      const findings = await handleTransaction(txEvent);
+
+      // assertions
+      expect(findings).toStrictEqual([]);
+    });
+
+    it('returns empty findings if contract address matches but not event', async () => {
+      const votingContract = (await votingAddressPromise).toLowerCase();
+
+      // logs data for test case: address match + no topic match
+      const logsNoMatchEvent = [
+        {
+          address: votingContract,
+          topics: [
+            ethers.utils.keccak256(ethers.utils.toUtf8Bytes('EncryptedVote(address,uint256,bytes32,uint256,bytes,bytes)')),
+            ethers.constants.HashZero,
```

Comment:
nit: I like to add comments to identify the topics just so it's clear what we're assigning HashZero to
e.g. for EncryptedVote:
ethers.constants.HashZero, // voter
ethers.constants.HashZero, // roundId
ethers.constants.HashZero, // identifier

Reply:
Done